doc/man/salt.7

```diff
@@ -54744,27 +54744,19 @@ supported by Salt Cloud. However, it may not necessarily be available on all
 Windows images.
 .SS Requirements
 .sp
-Salt Cloud makes use of \fIimpacket\fP and \fIwinexe\fP to set up the Windows Salt
+Salt Cloud makes use of \fIpypsexec\fP and \fIsmbprotocol\fP to set up the Windows Salt
 Minion installer.
 .sp
-\fIimpacket\fP is usually available as either the \fIimpacket\fP or the
-\fIpython\-impacket\fP package, depending on the distribution. More information on
-\fIimpacket\fP can be found at the project home:
+More information on pypsexec can be found at the project home:
 .INDENT 0.0
 .IP \(bu 2
-\fI\%impacket project home\fP
+\fI\%pypsexec project home\fP
 .UNINDENT
 .sp
-\fIwinexe\fP is less commonly available in distribution\-specific repositories.
-However, it is currently being built for various distributions in 3rd party
-channels:
+More information on smbprotocol can be found at the project home:
 .INDENT 0.0
 .IP \(bu 2
-\fI\%RPMs at pbone.net\fP
-.UNINDENT
-.INDENT 0.0
-.IP \(bu 2
-\fI\%openSUSE Build Service\fP
+\fI\%smbprotocol project home\fP
```

Comment:
This file does not need to be edited, it is generated when the package is built.